xmlvalidate/cmd/main.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/artefactual-sdps/temporal-activities/xmlvalidate"
+)
+
+func usage() {
+	fmt.Print(`
+USAGE: xmlvalidate [options] XMLFILE
+
+XMLFILE  Path of the XML file to be validated
+
+OPTIONS:
+  --xsd PATH  Path of an XSD file to validate against
+  --help      Display this help and exit
+
+`)
+}
+
+// To test: `go run ./cmd --xsd testdata/person.xsd testdata/person_valid.xml`
+func main() {
```

Comment:
We should try to follow https://artefactual.dev/docs/programming-languages/go/main/ and add some tests for this command.

Reply:
Hmm, okay. :+1: